doc/open-webui.md

```diff
@@ -0,0 +1,47 @@
+# Open WebUI
+
+[Open WebUI](https://github.com/open-webui/open-webui) is a user-friendly WebUI for LLMs (Formerly Ollama WebUI).
+
+{#start}
+## Getting Started
+
+```nix
+# In `perSystem.process-compose.<name>`
+{
+  services.open-webui."ow1".enable = true;
+}
+```
+
+## Examples
+
+{#ollama}
+### Open WebUI with ollama backend
+
+```nix
+{
+  services.open-webui."ow1" = {
+    enable = true;
+    environment.OLLAMA_API_BASE_URL = "http://localhost:11434";
```

Comment:
We should try to avoid hardcoding wherever we can (it can also be self-documenting):

```suggestion
    environment.OLLAMA_API_BASE_URL = let ollama = config.services.ollama.ollama1; in "http://${cfg.output.settings.environment.OLLAMA_HOST}";
```

(Untested code ^)

https://github.com/juspay/services-flake/blob/76f6081f51d8884b7d3196dbb45e8f8a11417a53/nix/ollama.nix#L110

Reply:
@shivaraj-bh This conversation is marked as resolved but the diff still shows `"http://localhost:11434"` for me?